src/styles/index.css

```diff
@@ -22,3 +22,7 @@ body {
 main {
   flex: 1;
 }
+
+.grecaptcha-badge {
+  display: none;
```

Comment:
This actually disables the spam checking:

https://stackoverflow.com/a/53986985